membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -13,19 +13,31 @@ import models._
 import monitoring.Metrics
 import play.api.libs.concurrent.Execution.Implicits._
 import play.api.libs.json.Json
-import play.api.mvc.{Controller, Result}
+import play.api.mvc._
 import play.filters.cors.CORSActionBuilder
 
-import scala.concurrent.Future
+import scala.concurrent.{Future, Promise}
 import scalaz.std.scalaFuture._
 import scalaz.syntax.std.option._
 import scalaz.{EitherT, \/}
+import configuration.Config.authentication
 
 
 class AttributeController extends Controller with LazyLogging {
 
+  def signed(): ActionBuilder[Request] = new ActionBuilder[Request] {
+    def invokeBlock[A](request: Request[A], block: (Request[A]) => Future[Result]) = {
+      if (request.headers.get("authentication").contains(authentication.key)) {
```

Comment:
Unless I'm mistaken, Identity uses the `Authorization` header. This header was originally used for Basic Auth. the use of `Bearer` is encouraged when you are providing a token (and indeed, specified in use cases like OAuth).

It looks like you've used `authentication`. I can't think of a standard where use of this header is specified so you can use whatever format you want. I'd recommend using `Authorization: Bearer token` for consistency, but if you want to stick to `authentication` it's probably worth at least capitalising the first letter to be consistent with other HTTP headers.